golem-worker-executor-base/src/services/mod.rs

```diff
@@ -158,6 +175,9 @@ pub struct All<Ctx: WorkerCtx> {
     template_service: Arc<dyn template::TemplateService + Send + Sync>,
     shard_manager_service: Arc<dyn shard_manager::ShardManagerService + Send + Sync>,
     worker_service: Arc<dyn worker::WorkerService + Send + Sync>,
+    worker_enumeration_service: Arc<dyn worker_enumeration::WorkerEnumerationService + Send + Sync>,
+    running_worker_enumeration_service:
+        Arc<dyn worker_enumeration::RunningWorkerEnumerationService + Send + Sync>,
```

Comment:
probably we should find out better naming - shorter one

Reply:
we can keep it for now :)